### PR TITLE
Update class-wpml-beaver-builder-translatable-nodes.php

### PR DESCRIPTION
**Pull request by vukvukovich**
* Add missed modules
* Update old ones
* Sorted all

### CODE REVIEW

**Comment by vukvukovich** [comment]
@andreasciamanna Fix for issue #1.
@amElnagdy It would be great if you can test this, even though I've tested.

File: src/class-wpml-beaver-builder-translatable-nodes.php [hunk 1]
```diff
@@ -237,102 +221,223 @@ public function initialize_nodes_to_translate() {
...
 						'field'       => 'success_url',
-						'type'        => __( 'Contact Form: Redirect link', 'wpml-string-translation' ),
+						'type'        => __( 'Contact Form: Redirect Link', 'wpml-string-translation' ),
 						'editor_type' => 'LINK'
 					),
 				),
 			),
-			'numbers'   => array(
-				'conditions' => array( 'type' => 'numbers' ),
+			'call to action' => array(
```
Comment (adrianosferreira) [inline src/class-wpml-beaver-builder-translatable-nodes.php]: @vukvukovich do you have sure that it has spaces?

**Comment by adrianosferreira** [comment]
@vukvukovich unit tests must be updated.

**Comment by andreasciamanna** [comment]
@vukvukovich please don't mention me: I might not be able to review your work and you would end up waiting forever for me.

PR should notify all WPML members, without the need of mentioning someone in particular (unless you need that specific person to run the review, that is):
image

**Comment by vukvukovich** [comment]
Sure, @adrianosferreira. Will update PR tomorrow. @andreasciamanna Ok. 😉

**Comment by amElnagdy** [comment]
@vukvukovich I've tested this and it's working fine.

**Comment by vukvukovich** [comment]
Thanks @amElnagdy!

**Comment by vukvukovich** [comment]
@adrianosferreira Done.

**Comment by adrianosferreira** [comment]
@vukvukovich it looks good to me. +1.

**Review by adrianosferreira — DISMISSED** [review]
Good, +1.

File: src/class-wpml-beaver-builder-translatable-nodes.php (hunk 1 above)
Comment (vukvukovich) [reply src/class-wpml-beaver-builder-translatable-nodes.php]: I've folded everything before commit with PHPStorm with WordPress support. Just checked once again and it looks fine. You can download and check.

Comment (adrianosferreira) [reply src/class-wpml-beaver-builder-translatable-nodes.php]: @vukvukovich I meant those spaces in the module key `call to action`. If it contains spaces, it is OK, just a doubt.

Comment (vukvukovich) [reply src/class-wpml-beaver-builder-translatable-nodes.php]: Hm.. I get it now, I've tested everything, but let me check once again.

**Review by strategio — DISMISSED** [review]
It looks good @vukvukovich +1

**Comment by vukvukovich** [comment]
@adrianosferreira @strategio I've tested it once more, it did not have any effect on the functionality. It was like that for who knows how long. Anyway, I've corrected it so we are following some standard.

Cheers!

**Pull request merged**